docs/api.md

```diff
@@ -371,7 +371,7 @@ Status: 200 OK
 Name | Type | Description
 ---|---|---
 owner | mongoose ObjectId | id of owner in user table.
-location | array of numbers | Location coordinates.
+location | array of numbers | Location coordinates, [ longitude, latitude ]. [Mongo's 2dsphere Index](https://docs.mongodb.com/manual/geospatial-queries/#geospatial-legacy)
```

Comment:
I don't think the links to mongo docs are necessary/desirable, because as far as I'm concerned, a reader of the docs doesn't need to even know that we are using mongo. Thoughts? (I would also suggest changing the type of id fields to string and not Mongoose Object Id at some point)

Reply:
yeah actually you're right I think